app/api/v1/api.py

```diff
@@ -1017,14 +971,57 @@ async def api_get_pool(
             status_code=status.HTTP_404_NOT_FOUND,
         )
 
+    pool_creator_clan = await clans_repo.fetch_one(id=pool_creator.clan_id)
+    pool_creator_clan_members: list[players_repo.Player] = []
+    if pool_creator_clan:
+        pool_creator_clan_members = await players_repo.fetch_many(
+            clan_id=pool_creator.clan_id,
+        )
+
     return ORJSONResponse(
         {
             "id": tourney_pool["id"],
             "name": tourney_pool["name"],
             "created_at": tourney_pool["created_at"],
-            "created_by": format_player_basic(pool_creator),
+            "created_by": {
+                "id": pool_creator.id,
+                "name": pool_creator.name,
+                "country": pool_creator.geoloc["country"]["acronym"],
+                "clan": (
+                    {
+                        "id": pool_creator_clan["id"],
+                        "name": pool_creator_clan["name"],
+                        "tag": pool_creator_clan["tag"],
+                        "members": len(pool_creator_clan_members),
+                    }
+                    if pool_creator_clan
+                    else None
+                ),
+                "online": pool_creator.is_online,
+            },
             "maps": {
-                f"{mods!r}{slot}": format_map_basic(bmap)
+                f"{mods!r}{slot}": {
+                    "id": bmap.id,
+                    "md5": bmap.md5,
+                    "set_id": bmap.set_id,
+                    "artist": bmap.artist,
+                    "title": bmap.title,
+                    "version": bmap.version,
+                    "creator": bmap.creator,
+                    "last_update": bmap.last_update,
+                    "total_length": bmap.total_length,
+                    "max_combo": bmap.max_combo,
+                    "status": bmap.status,
+                    "plays": bmap.plays,
+                    "passes": bmap.passes,
+                    "mode": bmap.mode,
+                    "bpm": bmap.bpm,
+                    "cs": bmap.cs,
+                    "od": bmap.od,
+                    "ar": bmap.ar,
+                    "hp": bmap.hp,
+                    "diff": bmap.diff,
+                }
```

Comment:
The pattern with the input var types for the `format_x_basic()` funcs was breaking -- I noticed they were only used for this function, so I decided it's not worth maintaining for v1 imo. I think we can rethink how to share formatting rules for v2.